### ISSUE TITLE
Diffing stuck at "Finding unmatched functions" in Ida 6.9

### ISSUE THREAD

**Issue by 821760408-sp** [joxeankoret/diaphora#155]
When trying to diff a 135M MIPS binary with another database export of a similar binary analyzed with Ida 7.1, using default settings, the process hangs at `find_unmatched()` every time, even though all the analyses seemed to have finished.

![screenshot 2019-02-19 09 23 11](https://user-images.githubusercontent.com/6172123/53021826-0d6b9000-3428-11e9-8410-787d89bedc01.png)

This is reproducable in Idaq and Idaq64 every time. Doing the same diff with reversed database exports in Ida 7.1 doesn't have the issue.

**Comment by joxeankoret** [comment]
Wow. IDA 6.9... From the log and the window I can see there, it seems to be some IDA bug. Please keep in mind that there is very little I can do with such an old version. That said, check [here](https://github.com/joxeankoret/diaphora/blob/a4472f9191ad5b6057bbc5483f1685b3ccf59762/diaphora_ida.py#L2141) if changing from this:

```
      if exported:
        log("Database exported. Took {} seconds".format(time.time() - t0))
```

to this:

```
      if exported:
        log("Database exported. Took {} seconds".format(time.time() - t0))
        hide_wait_box()
```

...fixes the issue. If it does, I will commit the change right away.

**Comment by 821760408-sp** [comment]
@joxeankoret yeah it does fix the issue. Tested with both 6.9 and 7.1
Yeah, migrating from Ida 6.9 to 7.x is not fun...

**Comment by joxeankoret** [comment]
Fixed with 1458893006727756c2563b45414c891451bac5b9
